sdk/servicebus/Azure.Messaging.ServiceBus/src/Amqp/AmqpReceiver.cs

```diff
@@ -363,7 +367,7 @@ private static void CloseLink(RequestResponseAmqpLink link)
                     message.Dispose();
                 }
 
-                return receivedMessages;
+                return receivedMessages ?? ReadOnlyList<ServiceBusReceivedMessage>.Empty;
```

Comment:
Do we need the new `ReadOnlyList` type, or can we just add the `Empty` instance as a `static readonly` member of this class?

Reply:
That would work too. I'm happy either way. I fell into the trap of seeing a util like namespace 😂

Reply:
If it were a fully functional list-type, I'd be all for it,  but I'm afraid the tempting name but single static member could be confusing.   It took me a moment to realize that I haven't been overlooking at built-in `ReadOnlyList` type for years.  :smile:

Reply:
Fixed